packages/shared/providers.yaml

```diff
@@ -5339,3 +5339,27 @@ falai:
             type: string
             title: API Key
             description: The API key for your fal.ai account
+
+acuity-scheduling:
+    display_name: Acuity Scheduling
+    categories:
+        - scheduling
+        - appointments
+    auth_mode: OAUTH2
+    authorization_url: https://acuityscheduling.com/oauth2/authorize
+    token_url: https://acuityscheduling.com/oauth2/token
+    connection_configuration:
+        - userId
+        - apiKey
+    post_connection_script: acuityPostConnection
+    webhook_routing_script: acuityWebhookRouting
+    proxy:
+        base_url: https://acuityscheduling.com/api/v1
+        decompress: true
```

Comment:
Are you certain decompress needs to be set? This is needed in rare unique cases

Reply:
haha tbh no I'm not sure, but I'll remove it for now!